test/extended/router/grpc-interop.go

```diff
@@ -48,7 +47,7 @@ var _ = g.Describe("[sig-network-edge][Conformance][Area:Networking][Feature:Rou
 	})
 
 	g.Describe("The HAProxy router", func() {
-		g.It("should pass the gRPC interoperability tests [apigroup:config.openshift.io][apigroup:route.openshift.io][apigroup:template.openshift.io]", func() {
+		g.It("should pass the gRPC interoperability tests [apigroup:config.openshift.io][apigroup:route.openshift.io]", func() {
```

Comment:
The test still uses the fixture in `router-shard.yaml` at line 314, so it seems premature to remove "[apigroup:template.openshift.io]" in this commit.  It isn't too important though if a subsequent commit removes the use of the fixture.